hapi-proto/src/main/java/com/hederahashgraph/fee/CryptoFeeBuilder.java

```diff
@@ -343,20 +340,12 @@ private int getCryptoTransferBodyTxSize(TransactionBody txBody) {
      * LONG_SIZE
      */
     int accountAmountCount = txBody.getCryptoTransfer().getTransfers().getAccountAmountsCount();
-    int cryptoTransfertBodySize = (BASIC_ACCT_AMT_SIZE) * accountAmountCount;
+    int cryptoTransfertBodySize = (BASIC_ACCOUNT_AMT_SIZE) * accountAmountCount;
     return cryptoTransfertBodySize;
   }
 
-  /*
-   * private long getTV(TransactionBody txBody) { long amount = 0; TransferList transferList =
-   * txBody.getCryptoTransfer().getTransfers(); List<AccountAmount> accountAmounts =
-   * transferList.getAccountAmountsList(); for (AccountAmount actAmt : accountAmounts) { if
-   * (actAmt.getAmount() > 0) { amount = amount + actAmt.getAmount(); } } return Math.round(amount /
-   * 1000); }
-   */
-
-  ////////////////////////////////////////////////////////////////////////// Query Fee
-  ////////////////////////////////////////////////////////////////////////// //////////////////////////////////////////////////////////////////////////
+  ///////////////////////////////////////////////////////// Query Fee //////////////////////////////////////////////////
+  //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
These are rather inconsistent. Probally change to either a simple single line comment // or  multi-line /* ... */ comment.

Reply:
Yeah.. changing it to a single line comment. I think it is just put in place to denote the start of query fee calculation methods.